editor/src/core/shared/deep-equality.ts

```diff
@@ -0,0 +1,98 @@
+import { keepDeepReferenceEqualityIfPossible } from '../../utils/react-performance'
```

Comment:
Again this is importing from outside the `core/shared` folder. Please can you move this or the file it imports or _something_ to fix that

Reply:
See previous comment.